tests/test_view.py

```diff
@@ -26,7 +29,7 @@ def test_view_extends_without_dictionary_parameters():
 
     view = container.make('View')
 
-    assert view('test') == 'test'
+    assert view('test').rendered_template == 'test'
 
 
 def test_render_from_container_as_view_class():
```

Comment:
Ok so the problem of your tests not passing and because you are not loading the container into any of the other `View()` classes. Look at line 25 in this file. This needs to be: `view = View().load_container(container)`

It should be like that in all tests (and the ViewProvider but I think you have it there)

Reply:
With this it was solved